src/lib/ui/Switch/Switch.tsx

```diff
@@ -0,0 +1,29 @@
+interface SwitchProps {
+  checked: boolean
+  onChange: (checked: boolean) => void
+  label?: boolean
+}
+
+function Switch({
+  checked,
+  onChange,
+  label = false,
```

Comment:
I missed this in the first review. This should be `disableLabel` since `label` implies the prop should be a text value. Alternatively, you can have `onLabel` and `offLabel` props, where if they are `null` they are hidden.